src/commcare_cloud/ansible/roles/keepalived/templates/notify_script.j2

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
+import smtplib
+import sys
+import hostname
+
+hostname = socket.gethostbyaddr(socket.gethostname())[0].split('.')[0]
+state= sys.arg[1]
+
+msg = MIMEMultipart()
+msg['From'] = "{{ sender_email }}"
+msg['To'] = "{{ recepient_email }}"
+msg['Subject'] = 'ALERT:proxy server status changed'
+
+body = "{0} entered in {1} state".format(hostname,state)
+msg.attach(MIMEText(body,'plain'))
+
+text = msg.as_string()
+server = smtplib.SMTP('{{ smtp_server }}',587)
+server.starttls()
+server.login("{{ sender_email }}", "{{ sender_password }}")
+server.sendmail("{{ sender_email }}", "{{ recepient_email }}",text)
+server.quit()
```

Comment:
seems like the email notifications can be done either via [keepalived](https://tecadmin.net/how-to-enable-email-alerts-in-keepalived/) iteslf or by using [postfix](https://docs.axway.com/bundle/APIGateway_753_ApplianceGuide_allOS_en_HTML5/page/Content/ApplianceTopics/WAI/keepalived_sendemail.htm)

Reply:
I tried that approach but keep getting this error.
`Error processing MAIL cmd on SMTP server  [74.125.68.108]:587 ,SMTP status code = 550`
Probably getting rejected due to SMTP considering it as SPAM.

Reply:
maybe try with the email settings we're already using (check `localsettings.py`)